gems/actionpack/6.0/actioncontroller.rbs

```diff
@@ -25,6 +25,8 @@ module ActionController
     include EtagWithTemplateDigest
     include EtagWithFlash
     include Caching
+    extend AbstractController::Caching::ClassMethods
+    extend AbstractController::Caching::Fragments::ClassMethods
```

Comment:
It would be better to not only extend ClassMethods but also include `AbstractController::Caching` here.

`ActionController::Caching` dynamically calls include inside the included block. Therefore it's better to include it explicitly.

https://github.com/rails/rails/blob/v6.0.6.1/actionpack/lib/action_controller/base.rb#L219
https://github.com/rails/rails/blob/v6.0.6.1/actionpack/lib/action_controller/caching.rb#L28-L30
https://github.com/rails/rails/blob/v6.0.6.1/actionpack/lib/abstract_controller/caching.rb#L46

Reply:
Thank you for your feedback, I fixed it in [6b169ec](https://github.com/ruby/gem_rbs_collection/pull/715/commits/6b169ec2f39abf9ee212ac721c9e2f2e6e416e13)